duffy/legacy/operations.py

```diff
@@ -2,17 +2,24 @@
 from starlette.status import HTTP_403_FORBIDDEN
 
 from ..configuration import config
+from .api_models import Credentials
 
 
-def obtain_credentials(apikey):
-    userlist = config["metaclient"]["usermap"]
+def obtain_destination():
     dest = (
         config["metaclient"]["dest"][0:-1]
         if config["metaclient"]["dest"][-1] == "/"
         else config["metaclient"]["dest"]
     )
```

Comment:
This is pretty complicated, why not `config["metaclient"]["dest"].rstrip("/")`?

Reply:
Hmm...

```
[t0xic0der@fedorable ~]$ python3
Python 3.10.1 (main, Dec  9 2021, 00:00:00) [GCC 11.2.1 20211203 (Red Hat 11.2.1-7)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> "hello."
'hello.'
>>> "hello/"
'hello/'
>>> a = "hello/"
>>> a.rstrip("/"
... )
'hello'
>>> "hello."
'hello.'
>>> b = "hello."
>>> b.rstrip("/")
'hello.'
>>> "hello/hello/hello/"
'hello/hello/hello/'
>>> c = "hello/hello/hello/"
>>> c.rstrip("/")
'hello/hello/hello'
>>> 
```

Point taken. Would make the change.